frontend/src/components/NotificationsCentral/index.tsx

```diff
@@ -1,7 +1,35 @@
 import React from 'react';
+import EmptySection from '../EmptySection';
+import './style.scss';
+import {
+  Button,
+  Row,
+} from '@carbon/react';
+import { Notification } from '@carbon/icons-react';
 
 const NotificationsCentral = () => (
-  <span> Placeholder </span>
+  <div className="notifications-space">
+    <Row>
+      <Button className='btn-viewed' kind="ghost">Mark all as viewed</Button>
+      <Button className='btn-all-notifications' kind="ghost">See all notifications</Button>  
+    </Row>
+    <hr></hr>
+    <div className="container">
+      <div className="vertical-center">
```

Comment:
These classes names are too generic, make them more specific to this notifications central section, like `notifications-central-container`